src/Components/App.tsx

```diff
@@ -1,13 +1,18 @@
 import React from 'react';
+import { Routes, Route } from 'react-router-dom'
 import Header from './Header';
 import Form from './Form';
 import '../Styles/App.css';
+import EventContainer from './EventContainer';
 
 const App = () => {
+
   return (
     <main className="App">
-      <Header />
-      <Form />
+      <Routes>
+        <Route path="/" element={<> <Header /> <Form /> </>} />
```

Comment:
did not know you could wrap Header and Form in a single path like this. Great call!